src/pages/Details.js

```diff
@@ -0,0 +1,100 @@
+import {useParams} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function Details({menusdata}) {
+  const params = useParams();
+  const currentmenu = menusdata.find(menu => menu.id === params.id);
+  return (
+    <DetailPage>
+      <Title>{currentmenu.name}</Title>
+      <MainPic src={currentmenu.image} alt={currentmenu.name} />
+      {currentmenu.category === 'Drinks' ? (
+        <></>
+      ) : (
+        <>
+          <DesContainer>
+            <Section>
+              <h2>Taste: </h2>
+              {currentmenu.taste}
+            </Section>
+            <Section>
+              <h2>Spiciness:</h2>
+              {currentmenu.spiciness} / 5
+            </Section>
+          </DesContainer>
+          <Section>
+            <h2>Ingredients:</h2>
+            <div>{currentmenu.ingredients}</div>
+          </Section>
+          <Section>
+            <h2>Review:</h2>
+            {currentmenu.reviews.map(review => {
+              return (
+                <SingelReview key={review.id}>
+                  {review.image ? <ReviewImg src={review.image} alt={currentmenu.name} /> : <></>}
+                  <div>
+                    {review.guest}: {review.text}
+                  </div>
+                </SingelReview>
+              );
+            })}
+          </Section>
+        </>
+      )}
```

Comment:
This conditional rendering is a bit confusing. Since you only want to render one thing, I would recommend to use the logical AND (`&&`) instead. And also update the condition.
```suggestion
      {currentmenu.category !== 'Drinks' && (
        <>
          <DesContainer>
            <Section>
              <h2>Taste: </h2>
              {currentmenu.taste}
            </Section>
            <Section>
              <h2>Spiciness:</h2>
              {currentmenu.spiciness} / 5
            </Section>
          </DesContainer>
          <Section>
            <h2>Ingredients:</h2>
            <div>{currentmenu.ingredients}</div>
          </Section>
          <Section>
            <h2>Review:</h2>
            {currentmenu.reviews.map(review => {
              return (
                <SingelReview key={review.id}>
                  {review.image ? <ReviewImg src={review.image} alt={currentmenu.name} /> : <></>}
                  <div>
                    {review.guest}: {review.text}
                  </div>
                </SingelReview>
              );
            })}
          </Section>
        </>
      )}
```

Reply:
Ok, I can rewrite it.